src/pdn/src/via.cpp

```diff
@@ -1272,6 +1501,20 @@ void ViaGenerator::determineRowsAndColumns(bool use_bottom_min_enclosure,
              rows,
              cols);
 
+  auto determine_enclosure = [this](bool use_min, bool is_x, int min_enc, int overlap_enc, const Constraint& contraint) -> int {
+    if (use_min) {
+      return min_enc;
+    } else {
```

Comment:
why not 'else if' - I don't see the need for the new scope

Reply:
fixed